deploy/configure-assets-after-deployment.ts

```diff
@@ -30,20 +38,38 @@ export const deploy: DeployFunction = async (environment) => {
   // const signer = provider.getSigner(
   //   '0x582a885C03A0104Dc3053FAA8486c178e51E48Db'
   // );
+
   const [signer] = await hre.getSigners();
   const market = await getMarket({ hre, signer });
   const certificate = await getCertificate({ hre, signer });
   const rNori = await getRestrictedNORI({ hre, signer });
   const removal = await getRemoval({ hre, signer });
   const bpNori = await getBridgedPolygonNori({ hre, signer });
+
+  // SW: Leaving the default local configuration as bridged polygon NORI
+  // for the purchase token to minimize test breakage.
+  let purchaseTokenAddress = bpNori.address;
+  let priceMultiple = BigNumber.from(100);
```

Comment:
@scottynomad why is this 100?

Reply:
Because that's the correct price multiple for NORI (price X 100).  I didn't push the usdc + 20x multiplier through all the tests because it broke too many of them and this is still the long termp lan.